reana-ui/src/pages/signin/Signin.js

```diff
@@ -59,9 +59,21 @@ export default function Signin() {
       <Segment>
         {config.cernSSO && (
           <>
-            <Button basic fluid size="large" onClick={handleClick}>
+            <Button basic fluid size="large" onClick={() => handleClick("cern_openid")}>
               Sign in with CERN Single Sign-On
             </Button>
+            {(config.localUsers || config.keycloakSSO) && (
+              <Divider section horizontal>
+                or
+              </Divider>
+            )}
+          </>
+        )}
+        {config.keycloakSSO && (
+          <>
+            <Button basic fluid size="large" onClick={() => handleClick("keycloak")}>
+              Sign in with {config.keycloakTitle} Single Sign-On
```

Comment:
Maybe we should allow administrators to customize the whole text?  For example, they might want to show `Sign in` or `Sign in with ESCAPE IAM`. /cc @tiborsimko

Reply:
Allowing just the `title` to be changed would give more certainty to the layout of the text I'd say. But you guys decide 😃

Reply:
For now, I guess it is enough to show the login provider name. It should be enough to boot. 

(And, if a need arises to display more text, and if we go for the `login` provider list in the Helm chart as discussed in the other PR, then we could simply add a new "login_button_text" or some such amongst properties there, should this need really arise.)

BTW see also the styling of the Zenodo login, using also simple layout with just login provider name changed:

![zenodo-login](https://user-images.githubusercontent.com/517546/220628127-5f916822-732f-467e-b52b-19a3b7b40dba.png)

The "or" separator is between external and internal login techniques... But for REANA, we would typically use only SSO or only Keycloak or only Local, so there would be typically only one button. (And later down the line, we could deprecate CERN SSO and move to CERN Keycloak, and even later down the time line, we could even deprecate Local and use only Keycloak, so that REANA would be single-user out-of-the-box , and multi-user when somebody adds Keycloak.)